plugin-veracode-dast/src/main/java/com/tracelink/prodsec/plugin/veracode/dast/service/VeracodeDastClientConfigService.java

```diff
@@ -42,14 +42,14 @@ public VeracodeDastClientConfigModel getClientConfig() {
 	 * @return an api client, pre-configured with the current config. Or null if no
 	 *         config exists
 	 */
-	public ApiClient getApiClient() {
+	public VeracodeApiClient getApiClient() {
 		VeracodeDastClientConfigModel config = getClientConfig();
 		if (config == null) {
 			return null;
 		}
-		ApiClient client = new ApiClient();
-		client.setConfig(config);
-		return client;
+		VeracodeApiClient apiClient = new VeracodeApiClient("https://api.veracode.com", config.getApiId(),
```

Comment:
Should this just be hardcoded in the client with the other strings?

Reply:
I think it's generally a good practice to introduce your base url. plus it makes testing very simple